src/main/java/com/ing/data/cassandra/jdbc/CassandraResultSet.java

```diff
@@ -373,6 +375,21 @@ public InputStream getAsciiStream(final int columnIndex) throws SQLException {
         }
     }
 
+    public Array getArray(int columnIndex) throws SQLException {
```

Comment:
`getArray(String columnLabel)` could also easily implemented.

Also, to be consistent with the rest of the code:
- the annotation `@Override` should be present on the method as an implementation of `java.sql.ResultSet` interface.
- the method argument should be `final`